airbyte/mcp/cloud_ops.py

```diff
@@ -686,6 +687,189 @@ def permanently_delete_custom_source_definition(
     )
 
 
+def update_cloud_source_config(
+    source_id: Annotated[
+        str,
+        Field(description="The ID of the deployed source to update."),
+    ],
+    *,
+    name: Annotated[
+        str | None,
+        Field(
+            description="Optional new name for the source.",
+            default=None,
+        ),
+    ] = None,
+    config: Annotated[
+        dict | str | None,
+        Field(
+            description="Optional new configuration for the source connector.",
+            default=None,
+        ),
+    ] = None,
+    config_secret_name: Annotated[
+        str | None,
+        Field(
+            description="The name of the secret containing the configuration.",
+            default=None,
+        ),
+    ] = None,
+) -> str:
+    """Update a deployed source connector's configuration on Airbyte Cloud.
+
+    This is a destructive operation that can break existing connections if the
+    configuration is changed incorrectly. Use with caution.
+
+    By default, the `AIRBYTE_CLIENT_ID`, `AIRBYTE_CLIENT_SECRET`, `AIRBYTE_WORKSPACE_ID`,
+    and `AIRBYTE_API_ROOT` environment variables will be used to authenticate with the
+    Airbyte Cloud API.
+    """
+    try:
```

Comment:
Don't use blanket try/except in the MCP tools. Let the failure be raised organically, unless it is an expected+handled error.